astropy/units/quantity.py

```diff
@@ -1332,6 +1332,33 @@ def _decompose(self, allowscaledunits=False, bases=[]):
     # https://numpy.org/doc/stable/reference/arrays.ndarray.html#array-conversion
 
     def item(self, *args):
+        """Copy an element of an array to a scalar Quantity and return it.
+
+        Parameters
+        ----------
+        *args
+            - none: in this case, the method only works for arrays with one
+              element (``a.size == 1``), which element is copied into a
+              Quantity scalar object and returned.
+            - int_type: this argument is interpreted as a flat index into the
+              array, specifying which element to copy and return.
+            - tuple of int_types: functions as does a single int_type
+              argument, except that the argument is interpreted as an nd-index
+              into the array.
+
+        Returns
+        -------
+        z : `Quantity`
+            A copy of the specified element of the array as a suitable
+            Quantity scalar
+
+        Notes
+        -----
+        This is different from :meth:`~numpy.ndarray.item` in that it always
+        returns a `Quantity`, not a Python scalar. For the data type of the
+        internal object, see the Note section of the linked reference.
```

Comment:
Personally, I'd remove the "For the data type..." as it is no longer relevant (it always is the dtype of the original array). If you want to keep things simple, you could also replace the whole docstring with
```
"""Copy an element of an array to a scalar Quantity and return it.

Like :meth:`~numpy.ndarray.item` except that it always
returns a `Quantity`, not a Python scalar.
"""
```

Reply:
I replaced with your suggested simplified docstring. No more "int_type"

Reply:
✅